src/lib/Scenes/Show/Screens/MoreInfo.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Separator } from "@artsy/palette"
+import { MoreInfo_show } from "__generated__/MoreInfo_show.graphql"
+import React from "react"
+import { FlatList, NavigatorIOS, ViewProperties } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { TextSection } from "../Components/TextSection"
+
+interface Props extends ViewProperties {
+  navigator: NavigatorIOS
+  show: MoreInfo_show
+}
+
+interface State {
+  sections: Array<{
+    type: "opening-reception" | "press-release"
+    data: any
+  }>
+}
+
+export class MoreInfo extends React.Component<Props, State> {
+  state = {
+    sections: [],
+  }
+
+  componentDidMount() {
+    const { show: _show } = this.props
+    // FIXME: opening reception field does not exist, stubbed for now
+    const show = {
+      ..._show,
+      opening_reception: "Thursday, Oct. 18th, 6pm-8pm\n\nArtists in attendance. Please RSVP.",
```

Comment:
👍 Makes sense to stub this for now

We do actually have structured info for opening receptions — e.g. see [this query](https://metaphysics-production.artsy.net/?query=%7B%0A%20%20show(id%3A%20%22blue-lotus-gallery-the-way-we-were%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20events%20%7B%0A%20%20%20%20%20%20event_type%0A%20%20%20%20%20%20description%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20start_at%0A%20%20%20%20%20%20end_at%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D) — but it is not exposed as a distinct field in Metaphysics. Will add a ticket for that.

Reply:
To follow up — I clarified this with Jun.

Although the wireframe happens to only show **Opening Reception** here, this should in fact be a list of every event that is associated with the show. 

So we should actually iterate over e.g. the data in the `events` field of the [sample query](https://metaphysics-production.artsy.net/?query=%7B%0A%20%20show(id%3A%20%22blue-lotus-gallery-the-way-we-were%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20events%20%7B%0A%20%20%20%20%20%20event_type%0A%20%20%20%20%20%20description%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20start_at%0A%20%20%20%20%20%20end_at%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D). No further follow-up is needed in Metaphysics to enable this (until/unless we have to deal with the stub show version of this).

Reply:
Got it, thanks for the follow-up explanation @anandaroop. I'll create a ticket to track this task separately.